src/ensembl/src/content/app/entity-viewer/gene-view/components/transcripts-filter/TranscriptsFilter.scss

```diff
@@ -2,44 +2,35 @@
 @import 'src/content/app/entity-viewer/gene-view/styles/constants';
 
 .container {
-  display: grid;
-  grid-template-columns: $left_column $middle_column_left_gap auto;
-  padding: 9px 0;
-}
-
-.filterLabel {
-  grid-column: 1/2;
-  text-align: right;
-  padding-right: 2px;
-  cursor: pointer;
-}
-
-.chevron {
-  margin-left: 10px;
-  height: 12px;
-  width: 12px;
-  margin-bottom: -3px;
+  grid-area: 3/1/3/4;  
+  position: sticky;
+  height: 100%;
+  top: -5px;
+  z-index: 3;  
 }
 
 .filterBox {
   display: grid;
-  grid-column: 3;
-  grid-template-columns: 270px minmax(210px, auto) 20px;
+  grid-column: 4;
```

Comment:
I don't understand. There is no such grid-column. There is no grid on `filterBox`'s parent anyway.

Reply:
filterBox itself is a grid (display: grid) and has 3 columns for sort by, filter by and then the cross.

Reply:
The `grid-column` rule says: hey, put me in the specified column of my immediate parent that has `display: grid`. Since the immediate parent of `filterBox` does not have `display: grid`, this line is useless.

Reply:
Thats been removed now.